src/components/Tile.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function Tile({ image, description, title, children }) {
+    console.log(image, description, title, children)
+  return (
+    <section>
+        {image && <img src={image} alt={description} />}
+        <h2>{title}</h2>
+        {children}
+    </section>
+  );
+}
+
+export default Tile;
```

Comment:
Het werkt! Het is clean. Geen overbodige code.
Dit was heel snel na kijkwerk. 

Goed gedaan. Ga zo door.